extensions/wikia/Discussions/maintenance/SiteStatus.class.php

```diff
@@ -5,7 +5,6 @@
 
 class SiteStatus {
 	const PROD_SERVICE_URL = 'https://serivces.wikia.com';
```

Comment:
Hah, that's interesting since it was here a while. I wonder if anybody is watching this site status. ;)